lunavl/sdk/faceengine/engine.py

```diff
@@ -336,66 +406,103 @@ def createIndexBuilder(self, descriptorVersion: int = 0, capacity: int = 0) -> I
+            self._faceEngine.createHeadWearEstimator(launchOptions=launchOptions.coreLaunchOptions), launchOptions
+        )
 
-    def createFaceNaturalLightEstimator(self) -> FaceNaturalLightEstimator:
+    def createFaceNaturalLightEstimator(
+        self, launchOptions: Optional[LaunchOptions] = None
+    ) -> FaceNaturalLightEstimator:
         """
         Create a face natural light estimator.
         Returns:
             estimator
         """
-        return FaceNaturalLightEstimator(self._faceEngine.createNaturalLightEstimator())
+        launchOptions = _getLaunchOptions(launchOptions)
+        return FaceNaturalLightEstimator(
+            self._faceEngine.createNaturalLightEstimator(launchOptions=launchOptions.coreLaunchOptions), launchOptions
+        )
 
-    def createRedEyeEstimator(self) -> RedEyesEstimator:
+    def createRedEyeEstimator(self, launchOptions: Optional[LaunchOptions] = None) -> RedEyesEstimator:
         """
         Create a red-eye estimator.
         Returns:
             estimator
         """
-        return RedEyesEstimator(self._faceEngine.createRedEyeEstimator())
+        launchOptions = _getLaunchOptions(launchOptions)
+        return RedEyesEstimator(
+            self._faceEngine.createRedEyeEstimator(launchOptions=launchOptions.coreLaunchOptions), launchOptions
+        )
 
-    def createFisheyeEstimator(self) -> FisheyeEstimator:
+    def createFisheyeEstimator(self, launchOptions: Optional[LaunchOptions] = None) -> FisheyeEstimator:
         """
         Create a fisheye effect estimator.
         Returns:
             estimator
         """
-        return FisheyeEstimator(self._faceEngine.createFishEyeEstimator())
+        launchOptions = _getLaunchOptions(launchOptions)
+        return FisheyeEstimator(
+            self._faceEngine.createFishEyeEstimator(launchOptions=launchOptions.coreLaunchOptions), launchOptions
+        )
 
-    def createFaceDetectionBackgroundEstimator(self) -> FaceDetectionBackgroundEstimator:
+    def createFaceDetectionBackgroundEstimator(
+        self, launchOptions: Optional[LaunchOptions] = None
+    ) -> FaceDetectionBackgroundEstimator:
         """
         Create a face background estimator.
+
+        Args:
+            launchOptions: estimator launch options
         Returns:
             estimator
         """
-        return FaceDetectionBackgroundEstimator(self._faceEngine.createBackgroundEstimator())
+        launchOptions = _getLaunchOptions(launchOptions)
+        return FaceDetectionBackgroundEstimator(
+            self._faceEngine.createBackgroundEstimator(launchOptions=launchOptions.coreLaunchOptions), launchOptions
+        )
 
-    def createImageColorTypeEstimator(self) -> ImageColorTypeEstimator:
+    def createImageColorTypeEstimator(self, launchOptions: Optional[LaunchOptions] = None) -> ImageColorTypeEstimator:
         """
         Create a image color type estimator.
+
+        Args:
+            launchOptions: estimator launch options
```

Comment:
maybe it's better to remove the `Args` section than to add it to all the other `createSomething`?

Reply:
Add to all, it's not difficult for me